CODEOWNERS

```diff
@@ -203,19 +203,50 @@ Jenkinsfile.nightly @cilium/ci
 /pkg/serializer @cilium/agent
 /pkg/service @cilium/loadbalancer
 /pkg/sysctl @cilium/bpf
-/pkg/testutils/ @cilium/ci
+/pkg/testutils/ @cilium/ci-structure
 /pkg/tuple @cilium/bpf
 /plugins/cilium-cni/ @cilium/kubernetes
 /plugins/cilium-docker/ @cilium/docker
 /proxylib/ @cilium/proxy
 /README.rst @cilium/docs-structure
-/test/ @cilium/ci
-/test/bpf/ @cilium/ci @cilium/bpf
-/test/Makefile* @cilium/ci @cilium/build
-/tests/ @cilium/ci
-Vagrantfile @cilium/ci
+/test/ @cilium/ci-structure
+/test/Makefile* @cilium/ci-structure @cilium/build
+# Service handling tests
+/test/k8sT/external_ips.go @cilium/loadbalancer @cilium/ci-structure
+/test/k8sT/manifests/externalIPs/ @cilium/loadbalancer @cilium/ci-structure
+/test/k8sT/Services.go @cilium/loadbalancer @cilium/ci-structure
+/test/runtime/lb.go @cilium/loadbalancer @cilium/ci-structure
+# Datapath tests
+/test/bpf/ @cilium/bpf
+/test/k8sT/Bandwidth.go @cilium/bpf @cilium/ci-structure
+/test/k8sT/DatapathConfiguration.go @cilium/bpf @cilium/ci-structure
+/test/k8sT/Verifier.go @cilium/bpf @cilium/ci-structure
+/test/runtime/benchmark.go @cilium/bpf @cilium/ci-structure
+/test/runtime/connectivity.go @cilium/bpf @cilium/ci-structure
+# Policy tests
+/test/k8sT/PoliciesNightly.go @cilium/policy @cilium/ci-structure
+/test/k8sT/Policies.go @cilium/policy @cilium/ci-structure
+/test/k8sT/StressPolicy.go @cilium/policy @cilium/ci-structure
+/test/runtime/Policies.go @cilium/policy @cilium/ci-structure
+# Hubble/monitoring tests
+/test/k8sT/hubble.go @cilium/hubble @cilium/ci-structure
+/test/runtime/monitor.go @cilium/hubble @cilium/ci-structure
+# L7 proxy tests
+/test/k8sT/fqdn.go @cilium/proxy @cilium/ci-structure
+/test/k8sT/KafkaPolicies.go @cilium/proxy @cilium/ci-structure
+/test/runtime/cassandra.go @cilium/proxy @cilium/ci-structure
+/test/runtime/fqdn.go @cilium/proxy @cilium/ci-structure
+/test/runtime/kafka.go @cilium/proxy @cilium/ci-structure
+/test/runtime/memcache.go @cilium/proxy @cilium/ci-structure
+# Misc. tests
+/test/k8sT/Health.go @cilium/health @cilium/ci-structure
+/test/k8sT/cli.go @cilium/cli @cilium/ci-structure
+/test/runtime/kvstore.go @cilium/kvstore @cilium/ci-structure
+/test/k8sT/Identity.go @cilium/agent @cilium/ci-structure
```

Comment:
Unsure about that one as well. Is this okay with @cilium/agent?

Reply:
yes